_community/google-groups.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Google Groups - Managing List Membership
+layout: text-optimized-width
+
+---
+
+# {{ page.title }}
+
+This documents how to join and leave a Google Group via webpage and email.
+
+If you have any issues following the guides below please reach out to us at [support@osg-htc.org](mailto:support@osg-htc.org) with the email you would like subscribed and the list you would like to subscribe too.
+
+##### Contents
+
+- [Joining a list via webpage](#joining-a-list-via-webpage)
+- [Leaving a list via webpage](#leaving-a-list-via-webpage)
+- [Joining a list via email](#joining-a-list-via-email)
+- [Leaving a list via email](#leaving-a-list-via-email)
+
+## Joining a list via webpage
```

Comment:
Ok, so just a rant here: This is not a good way to write procedural documentation. The user just wants to join or leave. How should they know whether it is better to use a webpage or email? Now they are confused, and rage quit.

Focus always, always, always on the user’s goal, and help them accomplish their goal as painlessly as possible. So there should be one heading for joining. One heading for leaving. Can users do anything else meaningful? See a history of messages for groups that allow it? Maybe we should generalize the page a bit, although I will not complain if we focus for now on joining and leaving.